### PR TITLE
Update query to work with PG v10, include meta in serialization, respect settlement period

### PR DESCRIPTION
**Pull request by mslipper**

### CODE REVIEW

**Comment by mslipper** [comment]
Not quite ready for merge.

**Comment by mslipper** [comment]
This is ready for review now.

File: lib/channel_manager.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import log from './util/log'
 import ChannelContract from './channel_contract'
 import PaymentManager from './payment_manager'
 import Web3 = require('web3')
+import { MachinomyOptions } from '../index'
```
Comment (ukstv) [inline lib/channel_manager.ts]: Circular dependency for the win!

Comment (mslipper) [reply lib/channel_manager.ts]: My bad. That's what I get for letting my IDE generate imports.

**Comment by mslipper** [comment]
Tests are failing because Circle uses PG v9.

**Pull request merged**